### PR TITLE
Fix hot reloading null refs

### PR DESCRIPTION
**Pull request by protodeep**
This brings in

- connection improvements and struct marshalling in LeapCSharp from platform
- latest LeapC.dll's
- simple stopping of editor playback on hot reload

### CODE REVIEW

File: Assets/LeapC/Logger.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public static void Log(object message)
     {
 #if DEBUG
 #if UNITY_EDITOR
-                    UnityEngine.Debug.Log(message);
+                    //UnityEngine.Debug.Log(message);
```
Comment (Amarcolina) [inline Assets/LeapC/Logger.cs]: Any reason we are commenting out this log statement?

File: Assets/LeapC/StructMarshal.cs [hunk 2]
```diff
@@ -16,8 +16,7 @@ private class StructContainer {
     private static int _sizeofT = Marshal.SizeOf(typeof(T));
 
     /**
-     * Converts an IntPtr to a struct of type T.  Does not allocate any
-     * garbage unlike Marshal.PtrToStruct(IntPtr, Type).
+     * Converts an IntPtr to a struct of type T.
```
Comment (Amarcolina) [inline Assets/LeapC/StructMarshal.cs]: Any reason this comment was changed?  The lack of garbage is a pretty important aspect of its operation.

**Comment by Amarcolina** [comment]
Merging this in for now, since it has important changes.  I'll make an issue pointing back here to address the comments.  Everything works great!

File: Assets/LeapC/Logger.cs (hunk 1 above)
Comment (jo3w4rd) [inline Assets/LeapC/Logger.cs]: Probably just a spurious change done while debugging and not properly restored.

**Pull request merged**